pkgdb2/api/extras.py

```diff
@@ -158,18 +160,25 @@ def _vcs_acls_cache(out_format='text', eol=False, collection=None):
         eol=eol,
         collection=collection,
         oformat=out_format,
-        skip_pp=APP.config.get('PKGS_NOT_PROVENPACKAGER', None))
+        skip_pp=APP.config.get('PKGS_NOT_PROVENPACKAGER', None),
+        namespace=namespace)
     output = []
     if out_format == 'json':
-        output = {'packageAcls': packages,
-                  'title': 'Fedora Package Database -- VCS ACLs'}
+        output = packages
+        if 'rpms' in output:
```

Comment:
Can we get away without special-casing this in the pkgdb server?

What reads this JSON endpoint -- the script that generates ACLs for gitolite?  I presume this change is for backwards compatibility.  By special casing this here on the pkgdb side, we'll have to *also* add special-case logic to that ACL generation script.

Can we keep the namespacing in the JSON response (to the detriment of backwards compatibility), please, and keep this stretch of code ignorant of the namespace details?